templates/zerver/help/sign-in.md

```diff
@@ -0,0 +1,19 @@
+# Sign in to Zulip
+## Sign in Online
+1. Go to your organization's Zulip URL
+2. Enter your email id and password.
```

Comment:
This should be "email address", not "email id".  

I think this article should also mention that you can instead use options like "Login with Google" if they are enabled.